File: 1005/1005.dart
import 'dart:io';

void main() {
  double a = double.parse(stdin.readLineSync());
  double b = double.parse(stdin.readLineSync());
  print("MEDIA = ${((a * 3.5 + b * 7.5) / 11).toStringAsFixed(5)}");
}

File: 1005/1005.cs
using System;
class URI {
    static void Main(string[] args) {
        double a, b, r;

        a = double.Parse(Console.ReadLine());
        b = double.Parse(Console.ReadLine());
        r = (a * 3.5 + b * 7.5) / 11;

        Console.WriteLine("MEDIA = " + r.ToString("F5"));
    }
}